libraries/mediaviewer/impl/src/main/kotlin/io/element/android/libraries/mediaviewer/impl/gallery/ui/VideoItemView.kt

```diff
@@ -84,6 +85,14 @@ private fun VideoInfoRow(
     Row(
         modifier = modifier
             .fillMaxWidth()
+            .background(
+                brush = Brush.verticalGradient(
+                    colors = listOf(
+                        Color.White.copy(alpha = 0f),
+                        Color.White,
+                    )
+                )
+            )
```

Comment:
I will update this, it has to render correctly on light theme too.

Reply:
Why do we need this at all?

Reply:
It's a bg for the info (picto + duration) displayed above the video thumbnail.